src/retasc/product_pages_api.py

```diff
@@ -0,0 +1,134 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+"""
+Production Pages API
+"""
+
+from functools import lru_cache
```

Comment:
Probably ok to use just `cache` (instead of `lru_cache(maxsize=None)`).

Reply:
you mean 
```
from functools import cache

    @cache
    @staticmethod
    def releases_by_product_phase(
```
?

Reply:
yes